pushapkscript/docker.d/worker.yml

```diff
@@ -65,9 +65,18 @@ products:
               package_names: [ "org.mozilla.fennec_aurora" ]
               certificate_alias: 'fennec-nightly'
               google:
+                # TODO Change to "beta" once the Fenix Nightly migration is done
                 default_track: 'alpha'
                 service_account: { "$eval": "GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_AURORA" }
                 credentials_file: { "$eval": "GOOGLE_CREDENTIALS_FIREFOX_AURORA_PATH" }
+            fennec-beta:
+              package_names: [ "org.mozilla.firefox_beta" ]
+              certificate_alias: 'fennec-beta'
+              google:
+                # TODO Change to "production" once the Fenix Beta migration is done
+                default_track: 'alpha'
+                service_account: { "$eval": "GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_BETA" }
+                credentials_file: { "$eval": "GOOGLE_CREDENTIALS_FIREFOX_BETA_PATH" }
```

Comment:
This variable doesn't exist yet, we need to define it above, see my previous comments in `init_worker.sh`